frontend/src/components/print/print-react/components/picasso/Responsibles.jsx

```diff
@@ -5,15 +5,24 @@ import { View, Text } from '../../reactPdf.js'
 const fontSize = 8
 const initialFontSize = fontSize * 0.75
 
+function displayNameFor(campCollaboration) {
```

Comment:
We already have `campCollaborationDisplayName` in `/common/helpers` and `responsiblesCommaSeparated` in `/print/helpers`. Could we use any of these?

Reply:
Fixed in e031d55